client/keys/delete_test.go

```diff
@@ -53,8 +53,7 @@ func Test_runDeleteCmd(t *testing.T) {
 	ctx := context.WithValue(context.Background(), client.ClientContextKey, &clientCtx)
 
 	err = cmd.ExecuteContext(ctx)
-	require.Error(t, err)
-	require.EqualError(t, err, "blah.info: key not found")
+	require.NoError(t, err)
```

Comment:
Is there a way to check that the error was printed by the `cmd`?

Reply:
I added the check for err printed.